src/main/java/controller/BlackjackController.java

```diff
@@ -0,0 +1,128 @@
+package controller;
+
+import domain.CardDeck;
+import domain.GameManger;
+import domain.GameResult;
+import domain.TrumpCard;
+import domain.user.Dealer;
+import domain.user.Player;
+import domain.user.User;
+import java.util.List;
+import java.util.Map;
+import view.CardConverter;
+import view.InputView;
+import view.OutputView;
+import view.Parser;
+
+public class BlackjackController {
+    private final OutputView outputView;
+    private final InputView inputView;
+
+    public BlackjackController(OutputView outputView, InputView inputView) {
+        this.outputView = outputView;
+        this.inputView = inputView;
+    }
+
+    public void run() {
+        List<String> playerNames = Parser.parseStringToList(inputView.inputUsers());
+        List<Player> users = playerNames.stream().map(Player::new).toList();
+        GameManger gameManger = new GameManger(users, new Dealer(), new CardDeck());
+        distributionFirstCard(gameManger, playerNames);
+
+        additionalPlayerCard(gameManger, playerNames);
+        additionalDealerCard(gameManger);
+
+        createGameResult(gameManger, playerNames);
+        calculateGameResult(gameManger);
+
+        inputView.close();
+    }
+
+    private void distributionFirstCard(GameManger gameManger, List<String> playerNames) {
+        gameManger.firstHandOutCard();
+
+        displayOpenCard(gameManger.getDealer().getName(), gameManger.getDealer());
+        playerNames.forEach(playerName -> displayOpenCard(playerName, gameManger.findUserByUsername(playerName)));
+    }
+
+    private void additionalDealerCard(GameManger gameManger) {
+        User dealer = gameManger.getDealer();
+        while (!dealer.isImpossibleDraw()) {
+            dealer.drawCard(gameManger.handOutCard());
+            outputView.displayDealerAddCard();
+        }
+    }
+
+    private void additionalPlayerCard(GameManger gameManger, List<String> playerNames) {
+        playerNames.forEach(playerName ->
+                addCardAllPlayer(gameManger, playerName));
+    }
+
+    private void calculateDealerResult(Map<User, GameResult> gameResult) {
+        int dealerLoseCount = getResultStateCount(gameResult, GameResult.WIN);
+        int dealerWinCount = getResultStateCount(gameResult, GameResult.LOSE);
+        int dealerDrawCount = getResultStateCount(gameResult, GameResult.DRAW);
+
+        outputView.displayDealerGameResult(dealerLoseCount, dealerWinCount, dealerDrawCount);
+    }
+
+    private int getResultStateCount(Map<User, GameResult> gameResult, GameResult status) {
+        return (int) gameResult.entrySet().stream()
+                .filter(entry -> entry.getValue() == status)
+                .count();
+    }
+
+    private void addCardAllPlayer(GameManger gameManger, String playerName) {
+        User player = gameManger.findUserByUsername(playerName);
+        while (!player.isImpossibleDraw()) {
```

Comment:
부정에 부정 조건은 한번에 읽기가 정말 어려운 조건이라고 생각해요. 

이런 경우에는 가능하면 긍정문으로 바꾸면 조금 더 조건을 파악하기가 쉬울 것 같아요. 😃 

```suggestion
        while (player.isDrawable()) {
```